image/templates/helm/stackrox-central/internal/defaults.yaml.htpl

```diff
@@ -225,11 +225,11 @@ defaults:
                 operator: DoesNotExist
       resources:
         requests:
-          memory: "1500Mi"
-          cpu: "1000m"
+          memory: "500Mi"
+          cpu: "1500m"
         limits:
-          memory: "3Gi"
-          cpu: "2000m"
+          memory: "3000Mi"
```

Comment:
Not to be *that* annoying, but this is technically less than before. Not really a big deal, though. Just commenting for awareness

Reply:
Moved all memory values to Gi.